keras/src/layers/compatibility_test.py

```diff
@@ -0,0 +1,281 @@
+import os
```

Comment:
My inclination is this isn't quite worth the maintenance cost, the tests added to KerasNLP just testing against different versions of Keras will buy us a lot, and simpler to maintain.

Reply:
Got it. This test has been removed.